integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -265,3 +269,113 @@ def get_address_from_coordinates(request, region_slug):
             "country": address.get("country"),
         }
     )
+
+
+def show_poi_form_ajax(request, region_slug, language_slug, poi_title):
+    """Render a POI form template
+
+    :param request: The current request
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :param language_slug: The slug of the current language
+    :type language_slug: str
+
+    :param poi_title: The title of the POI that the template defaults to
+    :type poi_title: str
+
+    :return: The html template of a POI form
+    :rtype: ~django.http.HttpResponse
+    """
+    poi_form = POIForm()
+    poi_translation_form = POITranslationForm(data={"title": poi_title})
+    language = get_object_or_404(Language, slug=language_slug)
+    url_link = f"{settings.WEBAPP_URL}/{region_slug}/{language_slug}/{poi_translation_form.instance.url_infix}/{poi_title}"
+
+    media_context_mixin = MediaContextMixin()
+    media_context_mixin.request = request
+    media_config_data = media_context_mixin.get_context_data()
+
+    return render(
+        request,
+        "events/_poi_form_widget.html",
+        {
+            **POIContextMixin().get_context_data(),
```

Comment:
```suggestion

    poi_context_mixin = POIContextMixin()
    poi_context_mixin.request = request

    return render(
        request,
        "events/_poi_form_widget.html",
        {
            **poi_context_mixin.get_context_data(),
```

Reply:
Yes, thank you! I'm afraid I did no proper testing after rebase 🙈